skmultilearn/model_selection/iterative_stratification.py

```diff
@@ -75,21 +75,25 @@
 import itertools
 from sklearn.utils import check_random_state
 
-def iterative_train_test_split(X, y, test_size):
+def iterative_train_test_split(X, y, test_size, random_state=None):
     """Iteratively stratified train/test split
 
     Parameters
     ----------
     test_size : float, [0,1]
         the proportion of the dataset to include in the test split, the rest will be put in the train set
-
+    
+    random_state : int
```

Comment:
I think the seed isn't necessarily of type int. it can be one of `None | int | instance of RandomState`, but I'm not 100% sure. Someone should chime in as well.
```suggestion
    random_state : None | int | instance of RandomState
```